build/proxysql.cnf

```diff
@@ -0,0 +1,30 @@
+admin_variables=
+{
+	admin_credentials="admin:admin;radmin:radmin"
+}
+
+# defines all the MySQL servers
+mysql_servers =
+(
+	{ address="host.docker.internal" , port=3306 , hostgroup=1, comment="Source" },
+	{ address="host.docker.internal" , port=3307 , hostgroup=2, weight=1000, comment="Readyset" },
+	{ address="host.docker.internal" , port=3306 , hostgroup=2, weight=1, comment="Source" }
+)
+
+mysql_variables=
+{
+	monitor_username="root"
+	monitor_password="noria"
+}
+
+mysql_users:
```

Comment:
the earlier entries in this file are followed by an equals sign, `=`; here you are following with a colon, `:`. Is that intentional?

Reply:
That is a good point. I got it from https://github.com/sysown/proxysql  README.
<img width="1103" alt="Screenshot 2024-09-03 at 12 54 03" src="https://github.com/user-attachments/assets/fc62bdfb-f18b-453e-ae98-6ae3783e2da4">

I can confirm that it is creating the users when I boot the containers:

```
$ mysql -h 127.1 -uradmin -pradmin -P 6032 -e "SELECT username, password from mysql_users"
mysql: [Warning] Using a password on the command line interface can be insecure.
+----------+----------+
| username | password |
+----------+----------+
| root     | noria    |
+----------+----------+
```